docs/exploration-and-organization/delete-and-restore.md

```diff
@@ -0,0 +1,136 @@
+| Alerts                             | Removed        | Removed                            | Not restored   |
+
+### Dashboards
+
+What happens to related items when you delete a dashboard?
+
+| Related item    | In Trash                   | Permanently deleted        | Restored |
+| --------------- | -------------------------- | -------------------------- | -------- |
+| Subscriptions   | Deactivated                | Deactivated                | Restored |
+| Custom homepage | Revert to default homepage | Revert to default homepage | Restored |
+
+### Model
+
+What happens to related items when you delete a model?
+
+| Related item                       | In Trash       | Permanently deleted                | Restored       |
+| ---------------------------------- | -------------- | ---------------------------------- | -------------- |
+| Question based on deleted question | Works normally | Breaks with `Card not found` error | Reactivated    |
+| Dashboard                          | Card removed   | Card removed                       | Card restored  |
+| Action                             | Works normally | Deleted                            | Works normally |
+
+## Collections
+
+What happens to related items when you delete a collection?
+
+| Related item                                   | In Trash | Permanently deleted | Restored |
+| ---------------------------------------------- | -------- | ------------------- | -------- |
+| All items and subcollections in the collection | In Trash | Permanently deleted | Restored |
+
+## Permanently deleting items
+
+Moving an item to Trash doesn't delete the item completely: you'll be able to restore the item from the Trash.
+
+To permanently delete an item:
+
+1. Go to Trash;
+2. Find the item you'd like to delete;
+3. Click on the checkbox next to the collection to select it;
+4. Select "Permanently delete". If you click this button, you won't be able to recover the item. It'll be lost to the void.
+
+## Deleting and restoring events and timelines
+
+Events and timelines can be archived and unarchived. See [Archiving Events and timelines](events-and-timelines.md#archiving-timelines).
+
+You won't see archived Events and Timelines in Trash. To see archived events and timelines, you need to [access them from the collection's page](events-and-timelines.md#view-archived-events-and-timelines).
+
+## Deleting and restoring SQL snippets
+
+SQL snippets can be archived and unarchived. See [Archiving SQL snippets](../questions/native-editor/sql-snippets.md#archiving-snippets).
+
+You won't see archived SQL snippets in Trash. To see archived SQL Snippets, you need to [access them from the Snippet menu](../questions/native-editor/sql-snippets.md#snippet-menu).
+
+## Deleting metrics and segments
+
+Metrics and events can be retired. See [Retiring Metrics and Segments](../data-modeling/segments-and-metrics.md#editing-and-retiring-segments-and-metrics).
```

Comment:
Hi @alexyarosh, this link is failing the `yarn run docs-lint-links` stage for me in CI.

Unclear to me what the link target should be. Did you forget to `git add` the `segments-and-metrics.md` file, or should the link maybe point to `segments.md` instead?

Reply:
Link to the CI failure for reference

https://github.com/metabase/metabase/actions/runs/11227506788/job/31245666789

Reply:
@appleby done #48478

Reply:
Ah makes, sense. Thanks!